add org_path documentation (#2109)
nf-core/tools: README.md

Note that a custom remote must follow a similar directory structure to that of `nf-core/moduleś` for the `nf-core modules` commands to work properly.

> The directory where modules are installed will be prompted or obtained from `org_path` in the `.nf-core.yml` file if available. If your modules are located at `modules/my-folder/TOOL/SUBTOOL` your `nf-core.yml` should have:

Reviewer: Most pedantic suggestion

```suggestion
The directory where modules are installed will be prompted or obtained from `org_path` in the `.nf-core.yml` file if available. If your modules are located at `modules/my-folder/TOOL/SUBTOOL` your `.nf-core.yml` should have:
```

> The directory where modules are installed will be prompted or obtained from the `.nf-core.yml` file if available. To add a directory name to your `.nf-core.yml` file, if your modules are located at `modules/my-folder/TOOL/SUBTOOL` you can add:

Reviewer 2: ```suggestion
The directory where modules are installed will be prompted or obtained from `org_path` in the `.nf-core.yml` file if available. If your modules are located at `modules/my-folder/TOOL/SUBTOOL` your `nf-core.yml should have:
```
